framework/doc/content/source/userobject/ActivateElementUserObject.md

```diff
@@ -0,0 +1,15 @@
+# ActivateElementUserObject
+
+This user object mimics the process of activating (adding) an element by moving the element from an "inactive" subdomain to the "active" subdomain. There are two metrics for activating an element, i.e., by +function path+ and by +variable value+.
```

Comment:
I'd prefer to se this refactored into an `ActivateElementUserObjectBase` and `ActivateElementsByPath` and `ActivateElementsCoupled`.

Reply:
emm okay.. I agree that would be better. I will work on making the change. Thanks for the suggestion!